packages/functions/src/common/BookAppointmentHelper.ts

```diff
@@ -103,3 +96,40 @@ function updateDonorAsync(donor: DbDonor, hospital: Hospital) {
 
   return setDonor(updatedDonor);
 }
+
+export function validateBookAppointment(
+  appointments: DbAppointment[],
+  donorId: string,
+  coordinatorId?: string,
+  donorDetails?: MinimalDonorDetailsForAppointment
+): ValidBookAppointmentResponse {
+  if (!appointments || appointments.length === 0) {
```

Comment:
Why do you check `!appointments`? It can't be undefined here (or the build would fail)

Reply:
Done